koji/flatten-el8/README.md

```diff
@@ -1 +1,8 @@
 This is a quick hack I put together from what was done by the epel folks in fedora infrastructure to build packages against modules
+
+Usage:  
+
+	`./test.sh `
```

Comment:
Based on the name of this script, I thought if I ran it things could just execute locally and not make changes over on Koji. Looking at that script it looks like the thing to run to generate updated repositories -- is that accurate? If so, could we rename to better capture that so I don't go and do dumb things :)

Reply:
Yeah, naming could use an update.  The script is intended to be an all in one command.

Reply:
You're mixing 2 styles to make it code in markdown leading to weird rendering:
```suggestion
	./test.sh
```

Reply:
Updated to a different name, and removed the back ticks.